src/components/CategoryMenu.tsx

```diff
@@ -70,7 +71,7 @@ const getCategoryItems = (): CategoryItem[] => {
   return Object.entries(CATEGORIES).map(([id, title]) => ({
     id: Number(id),
     title,
-    href: `/shop/category?categoryId=${id}`,
+    href: `${API_ENDPOINTS.SHOP_CATEGORY}${id}`,
```

Comment:
```suggestion
    href: {
        pathname: API_ENDPOINTS.SHOP_CATEGORY,
        query: {
              categoryId: id
          }
    },
```

이런식으로 객체형태로 하는게 더 직관적일 것 같아요